docs/stencil-docs/deploying-a-theme/theme-best-practices.md

```diff
@@ -0,0 +1,175 @@
+# Theme Development Best Practices Tutorial
+
+<div class="otp" id="no-index">
+
+### On this page
+- [Importing images](#importing-images)
+- [Injecting variables](#injecting-variables)
+- [Using Lighthouse](#using-lighthouse)
+- [Designing for accessibility](#designing-for-accessibility)
+- [Resources](#resources)
+</div>
+
+In this tutorial, you will learn the correct way to import theme images and inject theme variables. Also, theme developers will learn how to improve theme designs using [Lighthouse](https://developers.google.com/web/tools/lighthouse) and [accessibility best practices](https://developer.bigcommerce.com/stencil-docs/theme-accessibility).
+
+## Importing images
+There are two ways to import images. The quickest and easiest way to import images is to:
+1. Connect to your store using the [WebDAV](https://support.bigcommerce.com/s/article/File-Access-WebDAV) client of your choice.
+2. Add images to the `/content` directory.
+3. Reference images from there, as shown in the examples below:
+
+```json
+<img src="/content/image.jpg">
+```
+If you upload the image using a WebDAV client, you can access it within the theme using the `cdn` handlebar helper as follows:
+
+```json
+{{cdn "webdav:/img/image.jpg"}}
+```
+The WebDAV import option allows you to upload images in bulk. However, we suggest that you select the specific files needed and not import the entire markup file or unessential files as this can affect page load time.
+
+To learn about the BigCommerce folder structure best practices, see [Folder Structure](https://support.bigcommerce.com/s/article/File-Access-WebDAV#folder).
+
+The second way to import images is to use the image manager within the control panel.
+1. In the control panel, go to **Storefront** > **Image Manager**.
+2. Click **Upload images...**
+3. Click **Choose File** (or Browse) and select an image from your computer. Repeat for each additional image.
+4. Click **Upload**.
+
+When importing images, ensure images are appropriately-sized and have low byte size to increase a store's site speed. We recommend using a recent Stencil theme that uses responsive images. Once you have that, the best practice is to upload the best quality images you have, and BigCommerce will optimize them. We recommend JPEGs over other formats. See [Optimizing your Images](https://support.bigcommerce.com/s/article/Optimizing-Your-Images) for more details on how to optimize images.
+
+## Injecting variables
+
+Injecting javascript context variables allow you to access variables as needed to customize your theme. You can inject javascript context variables using the `inject` helper. The `inject` helper can be sent to the browser. Below is an example of an inject helper that can be added to your code.
```

Comment:
```suggestion
The `{{inject}}` Handlebars helper allows you to access store data through your theme by assigning a custom variable name to a property that you'd later like to return the value for. You can then use the `{{jsContext}}` Handlebars helper to access a stringified JSON object containing all injected data with your assigned variable name as the key in key-value pairs. The code example below uses `{{inject}}` and `{{jsContext}}` to log the product name to your browser's console when added to Cornerstone's `product.html` file:
```

Sorry for the delay. After playing with these helpers for a while, this is how I would explain the usage. Just a suggestion, though 👍